### ISSUE TITLE
How would we go about translating this to English?

### ISSUE THREAD

**Issue by astrohart** [PepperSniffer/GraphDISM#2]
I am just curious as to how one would obtain this code but with the GUI translated to English.

**Comment by PepperSniffer** [comment]
I built it when I was a student, I might go back and translate it properly :)

**Comment by Suleman-Elahi** [comment]
It is still not translated

**Comment by PepperSniffer** [comment]
Yeah, i never had the time to do it but feel free to contribute :)

**Comment by Dede333** [comment]
Just for contribution, you can go at https://github.com/dede333, there is two version VB# and C# of another DISM-GUI program and the code is translated in ENGLISH/FRENCH (i'm not a perfect translator, sorry).
Another think to do is to used StringBuilder for best optimization of memory.

Thanks and regards.
